BookingsApi/BookingsApi.UnitTests/Controllers/WorkHoursControllerTests.cs

```diff
@@ -541,5 +541,42 @@ public async Task UpdateVhoNonAvailabilityHours_With_Null_Hours_List_In_Request_
             
             _commandHandlerMock.Verify(x => x.Handle(It.IsAny<UpdateNonWorkingHoursCommand>()), Times.Never);
         }
+        
+        [Test]
+        public async Task DeleteVhoNonAvailabilityHours_Not_Valid_Id()
+        {
+            // Arrange
+            var username = "test.user@hearings.reform.hmcts.net";
```

Comment:
`username` isn't used here, can we remove it?

Reply:
removed